### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by JasonWeakley**
I would like to review this with whomever is doing the grading. I would like to go through the code and add comments to ensure that I fully understand what is happening in my code.

### CODE REVIEW

File: deselect.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+var CarLot = (function (newCarLot) {
+
+		newCarLot.deselectCard = function(element) {
+			var findClassOfClicked = document.getElementsByClassName("clicked");
+			// console.log("findClassOfClicked",findClassOfClicked);
+				for (var i = 0; i < findClassOfClicked.length; i++) {
+					findClassOfClicked.item(i).classList.remove("clicked");
+				}
+		},
+		newCarLot.styleCards = function(el) {
```
Comment (C-Stein) [inline deselect.js]: This function should accept an argument for the color of the border

File: quiz.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+													description +
+												closeCol +
+											closeRow;
+	};
+	writeToDOM.innerHTML = outputString;
+	
+	var putAllCardDivsInArray = document.getElementsByClassName("card");
+	 for (var i = 0; i < putAllCardDivsInArray.length; i++) {
+	 	var currentCard = putAllCardDivsInArray.item(i);
+	 	currentCard.addEventListener("click", function(event) {
```
Comment (C-Stein) [inline quiz.js]: any time you add an event listener, that functionality should be in the events.js IIFE

**Comment by C-Stein** [comment]
Functionality is all working- good job!!! There are two things you need to do in order to meet the technical requirements.nFirst- make sure  your function that expands the border thickness accepts two argments, as specified in the directions. Secondly, make sure ALL of your event listeners are located in your events.js IIFE. Let me know when  you're finished and keep up the good work.

**Comment by C-Stein** [comment]
Nice work! Go ahead and merge :+1:

**Comment by JasonWeakley** [comment]
Thanks!!

**Pull request merged**